arm-sql/2015-05-01-preview/swagger/virtualNetworkRules.json

```diff
@@ -0,0 +1,341 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Database management API provides a RESTful set of web APIs that interact with Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/virtualNetworkRules/{vnetFirewallRuleName}": {
+      "get": {
+        "tags": [
+          "VnetFirewallRule"
+        ],
+        "description": "Returns information about a Vnet Firewall Rule.",
+        "operationId": "VnetFirewallRule_Get",
```

Comment:
Sorry, I should clarify.  VnetFirewallRules_Get - we've been going plural_operation for operation id